rtc_base/BUILD.gn

```diff
@@ -19,6 +19,15 @@ if (!rtc_build_ssl) {
     assert(rtc_ssl_root != "",
            "You must specify rtc_ssl_root when rtc_build_ssl==0.")
     include_dirs = [ rtc_ssl_root ]
+
+    # WebRTC.Framework on iOS is a shared library. It checks symbol during building.
+    if (is_ios) {
+      libs = [
+        "crypto",
+        "ssl",
+      ]
+      lib_dirs = [ rtc_ssl_root + "/../lib" ]
```

Comment:
Is there any document on the requirement for directory layout of external ssl, when you specify external ssl for iOS build?

Reply:
It is the same layout as previous version. It looks like the layout description in readme.md is incorrect. I'll submit a separate PR in owt-client-native.